app/assets/v2/css/labs.css

```diff
@@ -0,0 +1,183 @@
+html {
+  overflow: hidden;
+}
+
+body {
+  height: 100vh;
+  color: var(--text-dark);  
+}
+
+.navbar {
+  flex-shrink: 0;
+}
+
+.navbar-nav > .img__spaceship {
+  display: none !important;
+}  
+
+.lab-hero-bg {
+  position: relative;
+  align-content: center;
+  height: 200px;
+}
+
+.lab-hero-bg-container{
+  position: absolute;
+  top:20px;
+  height: 500px;
+  width: 500px;
```

Comment:
how it looks in diff sizes can you post a mobile size print?

Reply:
Sure
![mobile](https://user-images.githubusercontent.com/21199234/49882897-90d09a00-fe57-11e8-97e0-9573aedf34a6.gif)